crates/e2e/src/tests/subblocks.rs

```diff
@@ -188,10 +190,11 @@ fn subblocks_are_included_with_failing_txs() {
             };
             let receipts = block.execution_outcome().receipts().first().unwrap();
 
-            // Assert that block only contains our subblock transactions and 3 system transactions
+            // Assert that block only contains our subblock transactions and system transactions
+            // (stablecoin exchange + subblocks signatures)
```

Comment:
nit: same thing here, this should be updated to only mention subblocks system txs